### DISCUSSION TITLE
Has anyone tried to remove Bootstrap and use Material UI or TailwindCss with abp?

### DISCUSSION THREAD

**Discussion by antosubash** [abpframework/abp#9394] [Q&A]
Most of the ABP Themes I have seen for abp are based on Bootstrap. Has anyone tried to remove bootstrap from UI and replace it with Material or Tailwind or any other css framework?  if yes can you share that repo or create a community article about this?

**Comment by realLiangshiwei** [comment]
Hi,

ABP is UI independent, you can use any UI framework. 

Someone  created the theme of layui, you can check this: https://github.com/colinin/Volo.Abp.AspNetCore.Mvc.UI.Theme.Layui

**Reply by antosubash** [reply]
Thanks for sharing the repo.
